### DISCUSSION TITLE
Feature: Add hooks like useColorMode & useBreakpoint

### DISCUSSION THREAD

**Discussion by xeinebiu** [chakra-ui/ark#3003] [Ideas]
Hello,

In ChakraUI, we have hooks to get some values from the active theme. Since colors and breakpoints are now defined in the Panda config, it might be useful to have hooks that provide access to the currently applied values, like the light/dark theme, breakpoints, etc. (a single source of truth).

Currently, I have custom hooks that use the same "breakpoint" values, but this leads to two places to maintain, one in the Panda config and the other in the custom hook.

Thanks in advance.

**Comment by segunadebayo** [comment]
I'd recommend these:

- Use a library like [`next-themes`](https://github.com/pacocoursey/next-themes) to handle color mode
- Use a library like [`use-hooks-ts`](https://usehooks-ts.com/react-hook/use-media-query) to handle media query changes

**Comment by segunadebayo** [comment]
Closing — these hooks are outside the scope of a headless component library.
